packages/components/components/elvis-modal/src/react/elvia-modal.tsx

```diff
@@ -43,21 +43,22 @@ export const Modal: FC<ModalProps> = function ({
   const [isModalOpen, setIsModalOpen] = useState(isShowing);
   const [fadeOut, setFadeOut] = useState(false);
 
+  const isModalOpenMemo = useMemo(() => [isModalOpen], [isModalOpen]);
   /** Get all slots */
   const { ref: modalHeading } = useSlot<HTMLHeadingElement>('heading', webcomponent, {
-    useEffectDependencies: isModalOpen,
+    useEffectDependencies: isModalOpenMemo,
   });
   const { ref: modalIllustration } = useSlot<HTMLDivElement>('illustration', webcomponent, {
-    useEffectDependencies: isModalOpen,
+    useEffectDependencies: isModalOpenMemo,
   });
   const { ref: modalPrimaryBtn } = useSlot<HTMLDivElement>('primaryButton', webcomponent, {
-    useEffectDependencies: isModalOpen,
+    useEffectDependencies: isModalOpenMemo,
   });
   const { ref: modalSecondaryBtn } = useSlot<HTMLDivElement>('secondaryButton', webcomponent, {
-    useEffectDependencies: isModalOpen,
+    useEffectDependencies: isModalOpenMemo,
   });
   const { ref: modalText } = useSlot<HTMLDivElement>('content', webcomponent, {
-    useEffectDependencies: isModalOpen,
+    useEffectDependencies: isModalOpenMemo,
```

Comment:
This started giving a type error, I assume it's because the types in `@types/react` have been improved. This is more correct